src/legacy/ui/public/agg_types/buckets/histogram.js

```diff
@@ -158,31 +159,30 @@ export const histogramBucketAgg = new BucketAggType({
       }
     },
 
+    {
+      name: 'extend_bounds',
+      default: false,
+      editorComponent: ExtendBoundsParamEditor,
+      write: () => {},
+    },
+
     {
       name: 'extended_bounds',
-      default: {},
-      editor: extendedBoundsTemplate,
+      default: {
+        min: '',
+        max: '',
+      },
+      editorComponent: ExtendedBoundsParamEditor,
       write: function (aggConfig, output) {
-        const val = aggConfig.params.extended_bounds;
+        const { min, max } = aggConfig.params.extended_bounds;
 
-        if (aggConfig.params.min_doc_count && (val.min != null || val.max != null)) {
-          output.params.extended_bounds = {
-            min: val.min,
-            max: val.max
-          };
+        if (aggConfig.params.extend_bounds &&
+          (min || min === 0) &&
+          (max || max === 0)) {
```

Comment:
I think you're looking for a check like "min is a number"? Lodash has `isNumber`

Reply:
There are two reasons why I do not prefer using `isNumber` here:
- we have plugins that write custom aggConfigs and I cannot be confident that these values are always numbers ;
- the recent @timroes message in slack:
![slack](https://user-images.githubusercontent.com/31325372/56950461-369e5000-6b3e-11e9-97ed-379ac9abe8b8.png)
so I'm not confident that `isNumber` would be a better solution.